generics/messageConstants/common.js

```diff
@@ -102,4 +102,8 @@ module.exports = {
   ADMIN:'admin',
   TENANT_ADMIN:'tenant_admin',
   ORG_ADMIN:'org_admin',
+  USER_PROFILE_KEY: 'userProfile.',
+  USER_PROFILE_KEY_OBS: 'observationInformation.userProfile.',
```

Comment:
@priyanka-TL check this might be need to be done for survey and surveySubmissions as well